README.md

```diff
@@ -1 +1,48 @@
-# woowa-writing-5
\ No newline at end of file
+# 이젠 글도 써? 🤷‍♂️
+
+어쩌면 모두가 느꼈을 감정이었을지도 모르겠다.  
+
+'아... 해야 하는 것도 많은데, 글은 또 무슨 글이야...'  
+'아... 나 글 못쓰는데...'  
+
+나 역시 마찬가지이다.  
+당장 레벨 1 마지막 미션에 온 우주의 기운을 모아 집중해도 모자랄 판에 글이라니...  
+그리 달갑지만은 않은 상태로 미션 레포를 열었다.  
+
+어떤 내용을 적어야 할지 잠시 고민해 보는데, 문득 생각이 든다.  
+'언제 한 달 반이 지나갔지...?'  
+
+정말 말도 안 되게 빠르게 지나갔다. 이유는 당연 몰입이 아닐까 싶다.  
+매일 8시에 시작한 하루는 한순간도 빠짐없이 프로그래밍으로 가득 차 있었고  
+밤 12시, 하루를 마무리하는 샤워 중에도, 심지어 꿈에서도 코드가 보였다.  
```

Comment:
[Dreams Come True](https://youtu.be/8uiR4SrDGZk)

Reply:
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ